packages/google-vertex/src/google-vertex-image-model.test.ts

```diff
@@ -313,5 +288,36 @@ describe('GoogleVertexImageModel', () => {
       );
       expect(result.response.modelId).toBe('imagen-3.0-generate-001');
     });
+
+    it('should only pass valid provider options', async () => {
+      prepareJsonResponse();
+
+      await model.doGenerate({
+        prompt,
+        n: 1,
```

Comment:
if this is the test from above, try to keep setting like n: 2 to the degree possible